yarn.lock

```diff
@@ -11683,30 +11722,20 @@ __metadata:
   linkType: hard
 
 "@typescript-eslint/utils@npm:^5.10.0, @typescript-eslint/utils@npm:^5.45.0":
-  version: 5.59.1
-  resolution: "@typescript-eslint/utils@npm:5.59.1"
+  version: 5.62.0
```

Comment:
I struggle to understand why this PR is bloated with dependency updates to ESLint packages. The diff is plenty large 😄

Reply:
I'm unsure why that happened as well... Seems to be resolved here: https://github.com/MetaMask/metamask-extension/pull/27782/commits/a1391244b4588908154be956fe8281830e71a895

Reply:
This was introduced again in a merge commit